File: 0075-sort-colors/solution.c
void sortColors(int* nums, int numsSize) 
{
     for(int i=0;i<numsSize;i++)
     {
        for(int j=0;j<numsSize;j++)
        {
            if(nums[i]<nums[j])
            {
                int temp=nums[i];
                nums[i]=nums[j];
                nums[j]=temp;
            }
        }
     }
    printf("[");
     for(int i=0;i<numsSize-1;i++)
     {
        printf("%d,",nums[i]);
     }
     printf("%d]",nums[numsSize-1]);

}

File: 0075-sort-colors/solution.java
class Solution {
    public void sortColors(int[] nums) {
        int n = nums.length;
        int r=0,w=0,b=0;
        for(int i=0;i<n;i++){
            if(nums[i]==0){
                r++;
            }else if(nums[i]==1){
                w++;
            }
            else {
                b++;
            }
        }
        int ind=0;
        while(r>0){
            nums[ind]=0;
            r--;
            ind++;
        }
        while(w>0){
            nums[ind]=1;
            ind++;
            w--;
        }
        while(b>0){
            nums[ind]=2;
            ind++;
            b--;
        }

    }
}
